app/components/auk/modal/body.hbs

```diff
@@ -1,5 +1,9 @@
 <div class="auk-modal__body">
-  <div class="{{unless @noPadding "auk-u-p-4"}}">
+  {{#unless @noPadding}}
```

Comment:
https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/rule/simple-unless.md

Reply:
@brenner-company Dat is voor deze. Volgens de linting rules mag een unless geen else hebben. Dan moet het een if hebben. Ik ging je het nog sturen  XD